webpack/assets/javascripts/react_app/pages/common/PageLayout/components/DocumentationButton.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Button, Icon } from 'patternfly-react';
+import PropTypes from 'prop-types';
+import packageJson from '../../../../../../../../package.json';
+import { translate as __ } from '../../../../common/I18n';
+
+const documentationUrl = (section = '', options = {}) => {
```

Comment:
IMO this function should live in a helper file so other places can reuse it.

Do we have other places in the code where we creates doc-url and can now migrate to use your logic?

Reply:
This is the only JS place. Moved to a helper file